### PR TITLE
Remove Fider stuff in the project

### PR DESCRIPTION
**Pull request by Kyubinhan**
Wanted to test `window.open('mailto:bcws.predictiveservice@gov.bc.ca')` but doesn't seem to be possible with Cypress

### CODE REVIEW

File: web/src/components/PageHeader.tsx [hunk 1]
```diff
@@ -60,15 +60,13 @@ export const PageHeader: React.FunctionComponent<Props> = ({ title }: Props) =>
...
         </div>
-        <a
-          className={classes.contact}
-          href={FIDER_LINK}
-          target="_blank"
-          rel="noopener noreferrer"
+        <div
           id="contact-link"
+          className={classes.contact}
+          onClick={() => window.open('mailto:bcws.predictiveservice@gov.bc.ca')}
```
Comment (andrea-williams) [inline web/src/components/PageHeader.tsx:66]:
```suggestion
          onClick={() => window.open('mailto:bcws.predictiveservices@gov.bc.ca')}
```

**Review by andrea-williams — APPROVED** [review]
Typo in the mailto link, but otherwise all good

File: web/src/components/PageHeader.tsx (hunk 1 above)
Comment (Kyubinhan) [reply web/src/components/PageHeader.tsx:66]: Ah I blame Sybrand for not having a consistent mail address in the ticket, nice catch!

Comment (andrea-williams) [reply web/src/components/PageHeader.tsx:66]: it's a good approach, always blaming @Sybrand . I hold him responsible for many of my failings and inadequacies

Comment (Sybrand) [inline web/src/components/PageHeader.tsx:66]: I think we should add a subject (maybe even a body?) - Something like "Predictive Services Unit - Website" - it's maybe too much trouble to add in "Percentile Calculator" or "Morecast" depending on where you are?
```suggestion
          onClick={() => window.open('mailto:bcws.predictiveservices@gov.bc.ca?subject=Predictive Services Unit - Website')}
```

**Review by Sybrand — APPROVED** [review]
Just a suggestion re. adding in a subject. At your discretion.

File: web/src/components/PageHeader.tsx (hunk 1 above)
Comment (Kyubinhan) [reply web/src/components/PageHeader.tsx:66]: Okay sounds good I will add the broader subject

Comment (Kyubinhan) [reply web/src/components/PageHeader.tsx:66]: Change of mind. Turns out it's fairly easy to be more specific

Comment (Sybrand) [reply web/src/components/PageHeader.tsx]: Always happy to take the blame.

**Pull request merged**